perses/annihilation/relative.py

```diff
@@ -2635,3 +2640,1611 @@ def _handle_exceptions(self):
+
+    For the second NonbondedForce, there is no alchemical or rest scaling. Long range correction is on.
+
+    This class can be tested using perses.tests.utils.validate_endstate_energies_point() and
+    perses.tests.utils.validate_endstate_energies_md(), as is done by perses.tests.test_relative.test_RESTCapableHybridTopologyFactory_energies
+
+    """
+    
+    # Constants copied from: https://github.com/openmm/openmm/blob/master/platforms/reference/include/SimTKOpenMMRealType.h#L89. These will be imported directly once we have addresssed https://github.com/choderalab/openmmtools/issues/522
+    M_PI = 3.14159265358979323846
+    E_CHARGE = (1.602176634e-19)
+    AVOGADRO = (6.02214076e23)
+    EPSILON0 = (1e-6*8.8541878128e-12/(E_CHARGE*E_CHARGE*AVOGADRO))
+    ONE_4PI_EPS0 = (1/(4*M_PI*EPSILON0))
+    #from openmmtools.constants import ONE_4PI_EPS0
+
+    _default_electrostatics_expression_list = [
+
+        "U_electrostatics;",
+
+        # Define electrostatics functional form
+        f"U_electrostatics = {ONE_4PI_EPS0} * chargeProd  * erfc(alpha * r_eff_electrostatics)/ r_eff_electrostatics;",
+
+        # Define chargeProd (with REST scaling)
+        "chargeProd = charge1 * electrostatics_rest_scale1 * charge2 * electrostatics_rest_scale2;",
+
+        # Define alpha
+        "alpha = {alpha_ewald};"
+
+        # Define r_eff
+        "r_eff_electrostatics = sqrt(r^2 + w_electrostatics^2);",
+
+        # Define 4th dimension terms
+        "w_electrostatics = w_scale * r_cutoff * (is_unique_old * lambda_alchemical_electrostatics_new + is_unique_new * lambda_alchemical_electrostatics_old);", # because we want w for unique old atoms to go from 0 to 1 and the opposite for unique new atoms
+        "is_unique_old = step(is_unique_old1 + is_unique_old2 - 0.1);", # if at least one of the particles in the interaction is unique old
+        "is_unique_new = step(is_unique_new1 + is_unique_new2 - 0.1);", # if at least one of the particles in the interaction is unique new
+        "w_scale = {w_scale};",
+        "r_cutoff = {r_cutoff};"
+
+    ]
+
+    _default_sterics_expression_list = [
+
+        "U_sterics;",
+
+        # Define sterics functional form
+        "U_sterics = 4 * epsilon * x * (x - 1.0);"
+        "x = (sigma / r_eff_sterics)^6;"
+
+        # Define sigma
+        "sigma = (sigma1 + sigma2) / 2;",
+
+        # Define epsilon (with rest scaling)
+        "epsilon = sterics_rest_scale1 * sterics_rest_scale2 * sqrt(epsilon_combined);",
+        "epsilon_combined = step(epsilon1 * epsilon2) * epsilon1 * epsilon2;", # if epsilon1 * epsilon2 < 0, sets epsilon_combined to 0
+
+        # Define r_eff
+        "r_eff_sterics = sqrt(r^2 + w_sterics^2);",
+
+        # Define 4th dimension terms
+        "w_sterics = w_scale * r_cutoff * (is_unique_old * lambda_alchemical_sterics_new + is_unique_new * lambda_alchemical_sterics_old);", # because we want w for unique old atoms to go from 0 to 1 and the opposite for unique new atoms
```

Comment:
@zhang-ivy : Just realized you might want this to be 
```python
        "w_sterics = w_scale * r_cutoff * (is_unique_old * (1-lambda_alchemical_sterics_old) + is_unique_new * (1-lambda_alchemical_sterics_new));",
```
if you want `lambda_alchemical_sterics_old` to go from 1 (old is on) to 0 (old is off)

Reply:
fixed!